x/feesplit/keeper/evm_hooks.go

```diff
@@ -64,7 +64,7 @@ func (k Keeper) PostTxProcessing(
 	}
 
 	txFee := sdk.NewIntFromUint64(receipt.GasUsed).Mul(sdk.NewIntFromBigInt(msg.GasPrice()))
-	developerFee := txFee.Mul(params.DeveloperShares.RoundInt())
+	developerFee := (params.DeveloperShares).MulInt(txFee).TruncateInt()
```

Comment:
Truncate and Round is different, pls use Round

Reply:
Done